packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergWebViewActivity.java

```diff
@@ -327,16 +327,16 @@ private void injectCssScript() {
         mWebView.evaluateJavascript(injectCssScript, message -> {
             if (message != null) {
                 String editorStyle = getFileContentFromAssets("gutenberg-web-single-block/editor-style-overrides.css");
-                editorStyle = removeWhiteSpace(removeNewLines(editorStyle));
```

Comment:
This rudimentary stripping of whitespace inadvertently modifies the CSS selectors, causing them to be interpreted as completely different selectors. E.g. `.ancestor .descendent` is different from `.sibling-1.sibling-2`. Ideally, a proper CSS magnification is utilized long term.